src/commands/logout.js

```diff
@@ -1,21 +1,25 @@
 // @flow
-import resetLogger from '../logger';
+import fs from 'fs';
 import { loginTokenExists, removeLoginToken } from '../utils/file';
+import { getCommands, resetLogger } from '../logger';
+import { promisify } from 'util';
+
+const copyFile = promisify(fs.copyFile);
 
 export const command = 'logout <username>';
 
 export const describe = 'log out of Sweesh account on this computer';
 
-export function handler(argv: any) {
-    resetLogger();
+export async function handler(argv: any) {
     const username = argv.username;
     const tokenExists = loginTokenExists(username);
-
     if (tokenExists) {
+        var promises = JSON.parse(getCommands())['commands'].map(command => copyFile(command['new_path'], command['old_path']));
```

Comment:
You have the right idea here, but there's 2 problems. 
1. We want to ensure we're doing it backwards (before the map, you can simply `.reverse()`. That's because we're treating the logger as a stack when we make changes (undoing = popping off the stack)
2. We have to do it sequentially backward. Because copyFile is async, the actual promises don't have to resolve in any particular order and it becomes non deterministic (but we need it to be in our case). So, we have to sequentially, do a for loop and `await` for 1 promise to resolve before kicking off the next one.